addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2759,6 +2762,10 @@ registry.BackgroundPosition = SnippetOptionWidget.extend({
             width: `${this.$target.innerWidth()}px`,
             height: `${this.$target.innerHeight()}px`,
         });
+
+        const $buttons = this.$overlayContent.find('div:last-child');
```

Comment:
Could you add a class on the div element and find it that way. `div:last-child` is not robust enough.

Reply:
done